recipe/gen_patch_json.py

```diff
@@ -673,6 +673,17 @@ def _gen_new_index(repodata, subdir):
                     _dep_parts[1] = _dep_parts[1] + ",<0.8.0"
                     record["depends"][i] = " ".join(_dep_parts)
 
+        # Integration between mdtraj and astunparse 1.6.3 on python 3.8 is
+        # broken, which was pinned for new builds in
+        # https://github.com/conda-forge/mdtraj-feedstock/pull/30 but should
+        # also be corrected on older builds
+        if (record_name == "mdtraj" and
```

Comment:
I think you need to limit this to a specific version else it will apply to all future ones:
```suggestion
        if (record_name == "mdtraj" and
            record["version"] == "1.9.5" and
```
(also regenerate the diff to make sure it's still correct)